### ISSUE TITLE
Core 2.5.0 Selected in Platformio.ini but Info Shows Core_Stage after flash ?

### ISSUE THREAD

**Issue by GasTurbineMan** [arendst/Sonoff-Tasmota#4866]
**IMPORTANT NOTICE**
If you do not complete the template below it is likely that your issue will not be addressed. When providing information about your issue please be as extensive as possible so that it can be solved by as little as possible responses.

**FAILURE TO COMPLETE THE REQUESTED INFORMATION WILL RESULT IN YOUR ISSUE BEING CLOSED**

**Describe the bug**
I compile my own builds using downloaded .zip and up until recently have always used core 2.3.0 however lately have been selecting 2.5.0 in the Platformio.ini file.  My issue is that after I build and upgrade the firmware .bin, the Info shows "Core/SDK Version | STAGE/3.0.0-dev(c0f7b44)"?
Is this correct?  I have tried on several devices and multiple builds.

_Also, make sure these boxes are checked [x] before submitting your issue - Thank you!_
- [x] _Searched the problem in issues and in the wiki_
- [x] _Hardware used_ : 
- [x] _Development/Compiler/Upload tools used_ : PlatformIO
- [ ] _If a pre-compiled release or development binary was used, which one?_ : n/a
- [ ] _You have tried latest release or development binaries?_ : n/a
- [x] _Provide the output of command_``status 0`` :
```
STATUS 0 OUTPUT HERE - DO NOT DELETE THE MARKERS ABOVE AND BELOW THIS LINE
18:16:58 MQT: stat/sonoff35/STATUS = {"Status":{"Module":1,"FriendlyName":["Dragon Flies"],"Topic":"sonoff35","ButtonTopic":"0","Power":1,"PowerOnState":3,"LedState":0,"SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":1}}
18:16:58 MQT: stat/sonoff35/STATUS1 = {"StatusPRM":{"Baudrate":115200,"GroupTopic":"sonoffs","OtaUrl":"http://192.168.85.60:6234/tasmoadmin/data/firmwares/sonoff.bin","RestartReason":"Software/System restart","Uptime":"0T00:14:22","StartupUTC":"2019-01-09T00:02:36","Sleep":50,"BootCount":117,"SaveCount":828,"SaveAddress":"FA000"}}
18:16:58 MQT: stat/sonoff35/STATUS2 = {"StatusFWR":{"Version":"6.4.1.8(sonoff)","BuildDateTime":"2019-01-08T17:58:11","Boot":31,"Core":"STAGE","SDK":"3.0.0-dev(c0f7b44)"}}
18:16:58 MQT: stat/sonoff35/STATUS3 = {"StatusLOG":{"SerialLog":0,"WebLog":2,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["mySSID",""],"TelePeriod":60,"SetOption":["01008129","55C180C0","00000001"]}}
18:16:58 MQT: stat/sonoff35/STATUS4 = {"StatusMEM":{"ProgramSize":484,"Free":516,"Heap":23,"ProgramFlashSize":1024,"FlashSize":1024,"FlashChipId":"14405E","FlashMode":3,"Features":["00000809","0F8A8684","04000000","00B404C6","000000C0"]}}
18:16:58 MQT: stat/sonoff35/STATUS5 = {"StatusNET":{"Hostname":"sonoff35-7898","IPAddress":"192.168.85.35","Gateway":"192.168.85.1","Subnetmask":"255.255.255.0","DNSServer":"192.168.85.90","Mac":"DC:4F:22:00:5E:DA","Webserver":2,"WifiConfig":5}}
18:16:58 MQT: stat/sonoff35/STATUS6 = {"StatusMQT":{"MqttHost":"192.168.85.96","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_455EDADA","MqttUser":"mymqttuser","MqttType":1,"MAX_PACKET_SIZE":1000,"KEEPALIVE":15}}
18:16:58 MQT: stat/sonoff35/STATUS7 = {"StatusTIM":{"UTC":"Wed Jan 09 00:16:58 2019","Local":"Tue Jan 08 18:16:58 2019","StartDST":"Sun Mar 10 02:00:00 2019","EndDST":"Sun Nov 03 03:00:00 2019","Timezone":99,"Sunrise":"07:35","Sunset":"17:29"}}
18:16:58 MQT: stat/sonoff35/STATUS10 = {"StatusSNS":{"Time":"2019-01-08T18:16:58","DS18B20":{"Temperature":50.5},"BMP280":{"Temperature":54.6,"Pressure":747.5},"PressureUnit":"mmHg","TempUnit":"F"}}
18:16:58 MQT: stat/sonoff35/STATUS11 = {"StatusSTS":{"Time":"2019-01-08T18:16:58","Uptime":"0T00:14:22","Vcc":3.477,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"POWER":"ON","Wifi":{"AP":1,"SSId":"mySSID","BSSId":"1C:B7:2C:C8:21:C0","Channel":6,"RSSI":100}}}
```

**To Reproduce**
_Steps to reproduce the behavior:_

**Expected behavior**
I would expect to see something more like:  Core/SDK Version | 2_5_0
-- | --

**Screenshots**
_If applicable, add screenshots to help explain your problem._

**Additional context**
This particular issue is not really causing me any problems, it just seems that something may not be quite correct?  I mean, am I using the 2.5.0 core or the Stage Core in my .bin file?

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by ascillato** [comment]
It is correct.

Arduino never released up to now the final version of 2.5.0

It is a beta at this moment, so that is why it said STAGE version. 

It is ok. Latest Arduino core for esp8266 is 2.4.2
Soon may be Arduino releases the final 2.5.0

**Comment by ascillato2** [comment]
Closing this issue as it has been answered.

Please, address general questions to the Tasmota Support Chat. Github issues are for bug reporting. Thanks 

### Support Information
See [Wiki](https://github.com/arendst/Sonoff-Tasmota/wiki) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
